CHANGELOG.md

```diff
@@ -6,14 +6,16 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
+<!-- Add new, unreleased changes here. -->
+
+## [3.0.0-pre.9] - 2017-01-25
```

Comment:
It's 2018 🎉 🎈 ! I do the same thing. Also maybe update the ones below :)

Reply:
OMG OMG OMG fix'd!

THANK YOU!